ts/src/binance.ts

```diff
@@ -3032,7 +3035,13 @@ export default class binance extends Exchange {
         } else if (market['inverse']) {
             response = await this.dapiPublicGetTicker24hr (this.extend (request, params));
         } else {
-            response = await this.publicGetTicker24hr (this.extend (request, params));
+            const rolling = this.safeValue (params, 'rolling', false);
+            params = this.omit (params, 'rolling');
+            if (rolling) {
+                response = await this.publicGetTicker (this.extend (request, params));
```

Comment:
Apparently this method doesn't exist on binance, what method was it supposed to be?

![](https://i.imgur.com/XAjMUna.png)